src/main/java/eu/execom/pomodoroTeam/entities/dto/TeamDto.java

```diff
@@ -6,25 +6,10 @@
 
 @Data
 public class TeamDto {
-    @JsonProperty("id")
-    private Long id;
 
     @JsonProperty("name")
```

Comment:
No need to use this annotation, since Jackson by default uses the field names to serialize an object into JSON.